.github/workflows/benchmark-runtime-weights.yml

```diff
@@ -95,7 +103,7 @@ jobs:
           echo "CHAIN=$chain" >> $GITHUB_ENV
 
       - name: Checkout codes on ${{ github.ref }}
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

Comment:
v4 doesn't work out during GHA. Because newer version has dependency issue. https://github.com/litentry/litentry-parachain/actions/runs/6420501769/job/17436254466